react/src/lib/components/DeckGLMap/layers/wells/wellsLayer.ts

```diff
@@ -348,3 +353,159 @@ function getLogProperty(
+
+    const t01 = Math.pow(dist_p0_p1, alpha);
+    const t12 = Math.pow(dist_p1_p2, alpha);
+    const t23 = Math.pow(dist_p2_p3, alpha);
+
+    const m1_x = (P2[0] - P1[0] + t12 * ((P1[0] - P0[0]) / t01 - (P2[0] - P0[0]) / (t01 + t12))); // eslint-disable-line
+    const m1_y = (P2[1] - P1[1] + t12 * ((P1[1] - P0[1]) / t01 - (P2[1] - P0[1]) / (t01 + t12))); // eslint-disable-line
+    const m1_z = (P2[2] - P1[2] + t12 * ((P1[2] - P0[2]) / t01 - (P2[2] - P0[2]) / (t01 + t12))); // eslint-disable-line
+
+    const m2_x = (P2[0] - P1[0] + t12 * ((P3[0] - P2[0]) / t23 - (P3[0] - P1[0]) / (t12 + t23))); // eslint-disable-line
+    const m2_y = (P2[1] - P1[1] + t12 * ((P3[1] - P2[1]) / t23 - (P3[1] - P1[1]) / (t12 + t23))); // eslint-disable-line
+    const m2_z = (P2[2] - P1[2] + t12 * ((P3[2] - P2[2]) / t23 - (P3[2] - P1[2]) / (t12 + t23))); // eslint-disable-line
+
+    const a_x = 2 * (P1[0] - P2[0]) + m1_x + m2_x;
+    const a_y = 2 * (P1[1] - P2[1]) + m1_y + m2_y;
+    const a_z = 2 * (P1[2] - P2[2]) + m1_z + m2_z;
+
+    const b_x = -3 * (P1[0] - P2[0]) - m1_x - m1_x - m2_x;
+    const b_y = -3 * (P1[1] - P2[1]) - m1_y - m1_y - m2_y;
+    const b_z = -3 * (P1[2] - P2[2]) - m1_z - m1_z - m2_z;
+
+    const c_x = m1_x;
+    const c_y = m1_y;
+    const c_z = m1_z;
+
+    const d_x = P1[0];
+    const d_y = P1[1];
+    const d_z = P1[2];
+
+    const x = a_x * ttt + b_x * tt + c_x * t + d_x;
+    const y = a_y * ttt + b_y * tt + c_y * t + d_y;
+    const z = a_z * ttt + b_z * tt + c_z * t + d_z;
+
+    return [x, y, z] as Position3D;
+}
+
+function splineRefine(data, refine: boolean) {
+    const ts = refine ? [0.2, 0.4, 0.6, 0.8] : [];
+
+    if (data["features"] === undefined) {
+        return;
+    }
+
+    const no_wells = data["features"].length;
+    for (let well_no = 0; well_no < no_wells; well_no++) {
+        const mds = data["features"][well_no]["properties"]["md"];
+
+        const coords = data["features"][well_no]["geometry"]["geometries"][1]["coordinates"]; // eslint-disable-line
+
+        const n = coords.length;
+        if (n < 3) {
+            continue;
+        }
+
+        // Point before first.
+        const x0 = coords[0][0] - coords[1][0] + coords[0][0];
+        const y0 = coords[0][1] - coords[1][1] + coords[0][1];
+        const z0 = coords[0][2] - coords[1][2] + coords[0][2];
+        const P_first: Position3D = [x0, y0, z0];
+
+        // Point after last.
+        const xn = coords[n - 1][0] - coords[n - 2][0] + coords[n - 1][0];
+        const yn = coords[n - 1][1] - coords[n - 2][1] + coords[n - 1][1];
+        const zn = coords[n - 1][2] - coords[n - 2][2] + coords[n - 1][2];
+        const P_n: Position3D = [xn, yn, zn];
+
+        //const md_first = 0.25 * (mds[0][0] - mds[0][1]) + mds[0][0];
+        //const md_n = 0.25 * (mds[0][n - 1] - mds[0][n - 2]) + mds[0][n - 1];
```

Comment:
Dead code? Either remove it or add a comment of what it does and when it should be enabled, if ever.